core/qtserialbluetooth.cpp

```diff
@@ -132,7 +132,7 @@ static dc_status_t qt_serial_open(void **userdata, const char* devaddr)
 #elif defined(Q_OS_ANDROID) || (QT_VERSION >= 0x050500 && defined(Q_OS_MAC))
 	// Try to connect to the device using the uuid of the Serial Port Profile service
 	QBluetoothAddress remoteDeviceAddress(devaddr);
-	serial_port->socket->connectToService(remoteDeviceAddress, QBluetoothUuid(QBluetoothUuid::SerialPort));
+	serial_port->socket->connectToService(remoteDeviceAddress, 1, QIODevice::ReadWrite | QIODevice::Unbuffered);
```

Comment:
Back when I wrote https://github.com/atdotde/subsurface/commit/57e83240ddb131d326d49afabc5b0ce208b6311d , @thiagomacieira concluded that this probably due to a bug in the other end:
http://lists.subsurface-divelog.org/pipermail/subsurface/2015-September/022054.html

It looks like the chips behave the same way when talking to a Mac.